src/overlays/actors/ovl_En_Horse_Link_Child/z_en_horse_link_child.c

```diff
@@ -73,41 +87,59 @@ static ColliderJntSphInit sJntSphInit = {
 
 static CollisionCheckInfoInit sColChkInfoInit = { 10, 35, 100, MASS_HEAVY };
 
-s32 D_808DFF10[] = { 1, 19 };
+// the animation frames that should have sfx
+static s32 sAnimSoundFrames[] = { 1, 19 };
 
-f32 D_808DFF18[] = { 1.0f, 1.0f, 1.5f, 1.5f, 1.5f };
+static f32 sAnimPlaySpeeds[] = { 1.0f, 1.0f, 1.5f, 1.5f, 1.5f };
 
 static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneScale, 1200, ICHAIN_STOP),
 };
 
-EnHorseLinkChildUnkFunc D_808DFF30[] = {
-    func_808DEA0C, func_808DED40, func_808DEB14, func_808DF194, func_808DF838, func_808DF620,
+typedef enum {
+    OOT_EPONA_ACTION_0,            // unfinished
```

Comment:
```suggestion
    /* 0 */ OOT_EPONA_ACTION_0,            // unfinished
```
Enum value comments please :)